torch/_inductor/codegen/cpp.py

```diff
@@ -187,17 +191,6 @@ def reduction_init(reduction_type, dtype):
     raise AssertionError(reduction_type)
 
 
-def reduction_init_vec(reduction_type, dtype):
```

Comment:
These vec reduction functions are moved to `CppVecKernel` so that we can generate the right vec_type according to the tiling factor.